.gitmodules

```diff
@@ -9,4 +9,4 @@
 	url = https://github.com/transmissions11/solmate
 [submodule "lib/openzeppelin-contracts"]
 	path = lib/openzeppelin-contracts
-	url = https://github.com/OpenZeppelin/openzeppelin-contracts
```

Comment:
I think their url does auto direct to the SnakeCase, so this maybe does seem more correct?